presto-hive-common/src/main/java/org/apache/hadoop/fs/PrestoExtendedFileSystemCache.java

```diff
@@ -19,6 +19,11 @@
     @Override
     protected FileSystem createPrestoFileSystemWrapper(FileSystem original)
     {
+        // Don't wrap LocalFileSystem otherwise FileSystem.getLocal(which
+        // do a type cast to LocalFileSystem) would fail
+        if (original instanceof LocalFileSystem) {
```

Comment:
It looks like your tests are failing because of this code? This is not related to the AliOss implementation as you mentioned it is being tracked in a different issue. Can you undo this?

Reply:
It is actually required, otherwise INSERT AliOss related tests would fail